tensorflow_datasets/core/download/kaggle.py

```diff
@@ -49,156 +49,92 @@
 
 KaggleType = collections.namedtuple(
     "KaggleType",
-    ["prefix", "download_cmd", "dl_flag", "extra_flag"])
+    ["prefix", "download_cmd", "dl_flag"])
 
 _KAGGLE_TYPES = {
     "dataset": KaggleType(
         prefix="dataset",
         download_cmd="datasets",
-        dl_flag="-d",
-        extra_flag="--unzip"),
+        dl_flag="-d"),
     "competition": KaggleType(
         prefix="competition",
         download_cmd="competitions",
-        dl_flag="-c",
-        extra_flag="")
+        dl_flag="-c")
 }
 
 
-def _get_kaggle_type(competition_name):
-  if "/" in competition_name:
+def get_kaggle_type(competition_or_dataset):
```

Comment:
Private functions should start with `_`, as previously (`_get_kaggle_type`). Same thing bellow.

Reply:
Fixed